src/AnimatedLineGraph.tsx

```diff
@@ -89,11 +89,11 @@ export function AnimatedLineGraph({
     })
 
     const previous = paths.current
-    let from: SkPath = previous.to ?? straightLine
+    let from: SkPath | null = previous.to ?? straightLine
```

Comment:
`straightLine` can never be `null`, so this can be removed

Reply:
`interpolate` can return null
interpolate type: `(method) SkPath.interpolate(end: SkPath, weight: number): SkPath | null`
so when you assign `interpolate` result to `from`, it can be `null`
https://github.com/margelo/react-native-graph/pull/31/files/00840df3ff16404aea00355f7092afada7c35c2d#diff-cfb3f852e9829282d769b73218a60d92f55779488387a3333b1c5717365419e9R94

https://github.com/Shopify/react-native-skia/pull/567

Reply:
on declaration, `from` doesn't need to be `SkPath | null`, but on second one, when we assign interpolate to from
```js
from = from.interpolate(previous.from, interpolateProgress.current)
```
it can be null
![image](https://user-images.githubusercontent.com/42688281/179746452-4760fc97-ada5-4119-8e7e-a2e79bedbdba.png)
![image](https://user-images.githubusercontent.com/42688281/179746836-c2489fd7-e24c-4794-bb8a-10067c0ffb26.png)
a possible solution is to create another variable but I don't see any difference

Reply:
I see, hmm this is a bit weird. We could do 

```ts
from = from.interpolate(previous.from, interpolateProgress.current) ?? from
```

But that's weird either way. I got your point now